wire-library/wire-tests/src/commonTest/proto/java/all_types.proto

```diff
@@ -114,8 +114,8 @@ message AllTypes {
   optional float default_float = 412 [default = 123.456e7 ];
   optional double default_double = 413 [default = 123.456e78 ];
   // Note: protoc doesn't allow some characters of the default value.
-  optional string default_string = 414 [default = "çok\a\b\f\n\r\t\v\1\01\001\17\017\176\x1\x01\x11\X1\X01\X11güzel" ];
-  optional bytes default_bytes = 415 [default = "çok\a\b\f\n\r\t\v\1\01\001\17\017\176\x1\x01\x11\X1\X01\X11güzel" ];
+  optional string default_string = 414 [default = "çok\a\b\f\n\r\t\v\1\01\001\17\017\176\x1\x01\x11\x01\x01\x11güzel" ];
+  optional bytes default_bytes = 415 [default = "çok\a\b\f\n\r\t\v\1\01\001\17\017\176\x1\x01\x11\x1\x01\x11güzel" ];
```

Comment:
Without this fix, `protoc` complains about syntax errors:
```
all_types.proto:117:102: Invalid escape sequence in string literal.
all_types.proto:117:105: Invalid escape sequence in string literal.
all_types.proto:117:109: Invalid escape sequence in string literal.
all_types.proto:118:100: Invalid escape sequence in string literal.
all_types.proto:118:103: Invalid escape sequence in string literal.
all_types.proto:118:107: Invalid escape sequence in string literal.
```

The "official" [spec](https://developers.google.com/protocol-buffers/docs/reference/proto2-spec#string_literals) is just wrong here. Hexadecimal literals can use either lower-case or upper-case "x", but hex byte escapes must use lower-case.